### PR TITLE
DEV-25766 add set and clear custom fields

### PR DESCRIPTION
**Pull request by george-haddad**
- Add example to set a custom field for a user
- Add example to clear a custom field for a user

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1340,83 +1340,141 @@ Returns information about the current user. See below for possible responses.
...
-            "status": 401
-            "type": "insufficient_privileges"
-          }
+            "error": {
+                "detail": "The user does not have the required privileges to perform the operation.",
+                "title": "Insufficient privileges",
+                "status": 401
+                "type": "insufficient_privileges"
+            }
+        }
```
Comment (rbarbey) [inline apiary.apib:1466]: From the comment and the values in the error hash, this should be a 403 response code, or not?

File: apiary.apib [hunk 2]
```diff
@@ -1642,18 +1700,19 @@ Retruns a list of all roles including the belonging privileges.
 
 + Response 401 (application/json)
 
-    // When the access token is missing or is invalid in the request
-    {
-        "error": {
+        // When the access token is missing or is invalid in the request
+        {
+            "error": {
             "detail": "Valid authentication has either not been provided or is insufficient.",
```
Comment (rbarbey) [inline apiary.apib:1706]: Here the indentation looks wrong (`detail` should be more indented than `error`) but maybe that's just a display problem.

**Comment by tkrisztian95** [comment]
Just a remark:
There are commits on this branch that should have done on the dev branch or in a different PR:
- fix white space formatting for linter
- modified package files from npm install command

File: apiary.apib (hunk 1 above)
Comment (george-haddad) [reply apiary.apib]: I was able to test and get a 403 response trying to update another user's custom fields that I did not have privileges to. Response code 403 but the payload was a 404 html/xml body.

**Review by rbarbey — APPROVED** [review]

**Pull request merged**